controllers/apply/form-router-next/lib/file-uploads.js

```diff
@@ -109,28 +126,50 @@ function uploadFile({ formId, applicationId, fileMetadata }) {
     });
 }
 
+function checkAntiVirusClamD(formId, applicationId, fileMetadata) {
+    const keyName = [formId, applicationId, fileMetadata.fileData.name].join('/');
```

Comment:
This `keyName` is the potential path on S3 only makes sense for the previous scan method. `fileMetadata.fileData.path` is the path we are interested in

Reply:
Sure. Would it be better if I use the `fileMetadata.fileData.name` instead as I'm not sure if metadata eg. `/var/folders/dn/844xbhwn2gqf37zpf7_1nv2h0000gn/T/upload_eacc44e2fb52ef868b9c3a84c22143b8` would be of any use ?

Reply:
Ah yeah good shout